### PR TITLE
Update k8s code to 1.14 and use general host executor

### LINKED ISSUE #59: Use general host executor
Why: simpler
How: https://github.com/kubernetes/kubernetes/pull/74391

### PR DESCRIPTION
**Pull request by cofyc**
- 0397c7d7455443571acc3a6f2fbd44adb5e13372 vendor files
- 2eba6f1dcbe67229f124d3b0c5ea3b310c4d3958 Update k8s code to 1.14 and use general host executor

fixes #59

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -12,33 +12,37 @@ required = [
...
 
 [[constraint]]
   name = "k8s.io/api"
-  branch = "release-1.13"
+  branch = "release-1.14"
 
 [[constraint]]
   name = "k8s.io/kubernetes"
-  branch = "release-1.13"
+  branch = "release-1.14"
```
Comment (msau42) [inline Gopkg.toml]: we could use the kubernetes-1.14.0 tags instead

Comment (cofyc) [reply Gopkg.toml]: done

**Pull request merged**